content/docs/reference-react.md

```diff
@@ -197,12 +197,11 @@ This API was introduced as a replacement of the deprecated `React.addons.cloneWi
 ```javascript
 React.createFactory(type)
 ```
+Retorna uma função que produz elementos React de um dado tipo. Assim como em [`React.createElement()`](#createElement), o argumento `type` pode ser uma *string* contendo o *tag name* (como, por exemplo, `'div'` ou `'span'`), um [componente React](/docs/components-and-props.html) (uma classe ou uma função), ou um [React *fragment*](#reactfragment).
```

Comment:
```suggestion
Retorna uma função que produz elementos React de um dado tipo. Assim como em [`React.createElement()`](#createElement), o argumento `type` pode ser uma *string* contendo a *tag name* (como, por exemplo, `'div'` ou `'span'`), um [componente React](/docs/components-and-props.html) (uma classe ou uma função), ou um [React *fragment*](#reactfragment).
```

Reply:
Here, I would change "de um dado tipo" to "de um determinado tipo" aswell.